tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -155,3 +158,37 @@ def test_autostop(self, cluster):
         # cluster.status()
         # # Check that last active is within the last second, so we know the activity wasn't just from the call itself
         # assert get_last_active() > time.time() - 1
+
+    @pytest.mark.level("minimal")
+    def test_logs_surfacing_scheduler_basic_flow(self, cluster):
+        if not cluster.den_auth:
+            pytest.skip(
+                "This test checking pinging cluster status to den, this could be done only on clusters "
+                "with den_auth that can be saved to den."
+            )
+
+        time.sleep(120)
```

Comment:
still not sure i follow why we need such a long sleep here? also could this be the reason why the CI tests are timing out?

Reply:
The cluster status check thread is executed two minutes after the thread creation. This enables the cluster to be initiated properly before the periodic status checks begin. However, since we are reusing a fixture, we may need to wait 2 mins. Side note: if we run the test separately for some reason (and not as a part of the test suite), the test might fail, since the thread has not been executed yet.

I think this test does not cause CI timeout; it is a minimal test, and therefore it is not running in CI (only in release testing). 

@jlewitt1